spec/backend/utils/resources-factory.spec.js

```diff
@@ -27,7 +27,7 @@ describe('ResourcesFactory', function () {
           resources() { return new Array(5) }
         }
         class Resource extends BaseResource {}
-        this.resourcesFactory = new ResourcesFactory([{ Database, Resource }])
+        this.resourcesFactory = new ResourcesFactory({}, [{ Database, Resource }])
```

Comment:
resources factory now accepts adminBro as a afirst argument - so now tests has to be adjusted